### PR TITLE
build: support yarn 1.19.x

### PR DESCRIPTION
**Pull request by IgorMinar**

### CODE REVIEW

**Comment by pkozlowski-opensource** [comment]
Yes! Thank you Igor!

**Review by petebacondarwin — APPROVED** [review]

**Review by devversion — APPROVED** [review]

File: aio/package.json [hunk 1]
```diff
@@ -175,4 +175,4 @@
     "xregexp": "^4.0.0",
     "yargs": "^7.0.2"
   }
-}
+}
```
Comment (gkalpak) [inline aio/package.json]: :cry:

File: package.json [hunk 2]
```diff
@@ -180,4 +180,4 @@
     "**/graceful-fs": "4.2.2"
   },
   "cldr-data-coverage": "full"
-}
+}
```
Comment (gkalpak) [inline package.json]: :cry:

**Review by gkalpak — APPROVED** [review]

File: aio/package.json (hunk 1 above)
Comment (IgorMinar) [reply aio/package.json]: clang-format? I'll revert that part.

File: package.json (hunk 2 above)
Comment (IgorMinar) [reply package.json]: fixed.

File: aio/package.json [hunk 3]
```diff
@@ -83,7 +83,7 @@
   "//engines-comment": "Keep this in sync with ../package.json",
   "engines": {
     "node": ">=10.9.0 <13.0.0",
-    "yarn": ">=1.17.3 <=1.19.1"
+    "yarn": ">=1.17.3 <1.20.0"
```
Comment (josephperrott) [inline aio/package.json]: Can we instead just move to `>=1.17.3 <2.0.0`?

**Review by josephperrott — CHANGES_REQUESTED** [review]

**Pull request closed without merging**